options.js

```diff
@@ -1,13 +1,12 @@
+const file_selector = document.getElementById("file-selector");
 
-const file_selector = document.getElementById("file-selector")
-
-class Options {
+class Options extends Screen {
```

Comment:
This seems to be generating an illegal constructor error that prevents the page from loading. I've looked around the code to see why exactly it is causing said error, but I have had no such luck so far.

Reply:
Where are you seeing this error?  At runtime in the dev console?

Reply:
Yes, it appears in the dev console while the page is loading, after which it stops the program.

Reply:
Hmm... I don't see it and it works for me.  Are you merging this branch with anyother branch when testing this?

Reply:
hm, I don't believe so, I'll try it again.

Reply:
Weird, it's working now. It was probably something in the version control that may have messed things up.